app/components/publications/new-publication-modal.js

```diff
@@ -60,4 +61,17 @@ export default class NewPublicationModal extends Component {
   async isPublicationNumberAlreadyTaken() {
     this.numberIsAlreadyUsed = await this.publicationService.publicationNumberAlreadyTaken(this.number, this.suffix);
   }
+
+  @action
```

Comment:
Deze method moet niet geannoteerd worden met `@action`. Ze wordt niet aangeroepen van in de template.

Reply:
Moet misschien ook geen aparte methode meer zijn?